exonum-java-binding/qa-service/src/main/java/com/exonum/binding/qaservice/ApiController.java

```diff
@@ -278,6 +289,29 @@ private void getActualConfiguration(RoutingContext rc) {
     respondWithJson(rc, configuration);
   }
 
+  private void getTime(RoutingContext rc) {
+    Optional<ZonedDateTime> time = service.getTime();
+    respondWithJson(rc, time.map(ApiController::convertZdtToString));
+  }
+
+  private void getValidatorsTimes(RoutingContext rc) {
+    Map<PublicKey, ZonedDateTime> validatorsTimes = service.getValidatorsTimes();
+    Map<PublicKey, String> stringValidatorsTimes = convertValidatorsTimesValues(validatorsTimes);
+    respondWithJson(rc, stringValidatorsTimes);
+  }
+
+  @VisibleForTesting
+  static Map<PublicKey, String> convertValidatorsTimesValues(
+      Map<PublicKey, ZonedDateTime> validatorsTimes) {
+    return validatorsTimes.entrySet().stream()
+        .collect(Collectors.toMap(Entry::getKey, t -> convertZdtToString(t.getValue())));
+  }
+
+  @VisibleForTesting
+  static String convertZdtToString(ZonedDateTime zonedDateTime) {
+    return DateTimeFormatter.ofPattern(DATE_TIME_FORMAT).format(zonedDateTime);
```

Comment:
How about to add `ZonedDateTime` json serializer to `common`? I had the same thing for parsing responses in LC

Reply:
It might make sense if it works with any ZDT. @MakarovS please create a separate task for that with references to code to update to ZDT serializer

Reply:
https://jira.bf.local/browse/ECR-2978